test/integration/supervisor_discovery_test.go

```diff
@@ -50,98 +50,100 @@ func TestSupervisorOIDCDiscovery_Disruptive(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Minute)
 	defer cancel()
 
-	temporarilyRemoveAllFederationDomainsAndDefaultTLSCertSecret(ctx, t, ns, defaultTLSCertSecretName(env), client, testlib.NewKubernetesClientset(t))
```

Comment:
I'm not sure if I understand this change. Won't the default TLS cert secret usually not exist on the cluster? If it were there, wouldn't we want to remove it so that this test can control its contents?